Loenn/triggers/lightningstrike.lua

```diff
@@ -0,0 +1,36 @@
+local lightning = {}
+
+lightning.name = "ContortHelper/LightningStrikeTrigger"
+lightning.placements = {
+    name = "default",
+    data = {
+        audioEvent = "event:/new_content/game/10_farewell/lightning_strike",
+        shake = 0.3,
+        amount = 2,
+        depth = -8500,
+        flashColor = "White",
+        flashColorAlpha = "0.6",

```

Comment:
This option is set to a string value. No idea if this is intended.
```suggestion
        flashColorAlpha = 0.6,

```